src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/trial_of_the_crusader.h

```diff
@@ -62,8 +62,8 @@ const Position ToCCommonLoc[]=
     {563.833008f, 195.244995f, 394.585561f, 0}, //  5 - Center
     {573.5f, 180.5f, 395.14f, 0},               //  6 Move 0 Right
     {553.5f, 180.5f, 395.14f, 0},               //  7 Move 0 Left
-    {573.0f, 170.0f, 395.14f, 0},               //  8 Move 1 Right
```

Comment:
Nop: https://gist.github.com/chaodhib/02508362678cf1e28505d885b74db5e4#file-twin-vaklyr-hover-analysis-txt

No waypoint seems to be blizz like for this boss it seems at the moment.